assessment.sql

```diff
@@ -5,24 +5,58 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  height NUMERIC(4,2),
+  weight NUMERIC(4,2),
+  dob DATE,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+  VALUES ('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black')
+;
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv)
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT * FROM licensees
+  WHERE eye_color = 'Hazel'
+    OR hair_color = 'Brown'
+    OR hair_color = 'Black'
+;
```

Comment:
The question asked for a query to retrieve all fields of all rows where `eye_color` is `'Hazel'` **and** `hair_color` is either `'Black'` or `'Brown'`. Otherwise, this is a good statement. 3/4